src/components/Apply.tsx

```diff
@@ -1,34 +1,43 @@
 import React from 'react';
 import { Helmet } from 'react-helmet';
+import { Container, Box, Typography } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-import { Container, Box, Paper } from '@material-ui/core';
 
```

Comment:
unnecessary line here

Reply:
Not exactly sure what you mean here?

Reply:
sorry that wasn't clear, we don't need the extra line between the import statements